### PR TITLE
Add Pressable Component

### PR DESCRIPTION
**Pull request by gedeagas**
This PR add Pressable Component

### CODE REVIEW

File: src/components/Pressable.re [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+[@bs.obj]
+external rippleConfig:
+  (~color: Color.t=?, ~borderless: bool=?, ~radius: float=?) => rippleConfig;
+
+[@react.component] [@bs.module "react-native"]
+external make:
+  (
+    ~ref: ref=?,
+    // Pressable props
```
Comment (MoOx) [inline src/components/Pressable.re:13]:
```suggestion
    // Accessibility props
```

File: src/components/Pressable.re [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    ~focusable: bool=?,
+    ~importantForAccessibility: [@bs.string] [
+                                  | `auto
+                                  | `yes
+                                  | `no
+                                  | [@bs.as "no-hide-descendants"]
+                                    `noHideDescendants
+                                ]
+                                  =?,
+    ~children: React.element=?,
```
Comment (MoOx) [inline src/components/Pressable.re:47]:
```suggestion
    // Pressable props
    ~children: React.element=?,
```

**Comment by MoOx** [comment]
Also I see that Pressable have a minPressDuration (float) props https://github.com/facebook/react-native/commit/4aaf62998247bcfd8ebf369d73290096fde08012

**Comment by gedeagas** [comment]
> Also I see that Pressable have a minPressDuration (float) props [facebook/react-native@4aaf629](https://github.com/facebook/react-native/commit/4aaf62998247bcfd8ebf369d73290096fde08012)

Altho Pressability has minPressDuration the Pressable component does not. So I don't put the props on Pressable component because currently there is no way to pass minPressDuration into Pressable component.

**Review by MoOx — COMMENTED** [review]
Since we don't have enough doc, I would appreciate if you could add minimal documentation with a tiny example in the markdown file please, that would be awesome.

File: src/components/Pressable.md [hunk 3]
```diff
@@ -3,3 +3,27 @@ id: components/Pressable
...
+Pressable is a Core Component wrapper that can detect various stages of press interactions on any of it's defined children.
+
+## How it works
+
+On an element wrapped by Pressable:
+
+1. onPressIn is called when a press is activated.
+2. onPressOut is called when the press gesture is deactivated.
+
+You can read more about this here.
```
Comment (MoOx) [inline src/components/Pressable.md:16]: Did you forget a link maybe?

**Review by MoOx — COMMENTED** [review]
Almost good! :)

File: src/components/Pressable.md (hunk 3 above)
Comment (gedeagas) [reply src/components/Pressable.md:16]: yep hahah. sorry about that.

**Comment by gedeagas** [comment]
Updated @MoOx.

File: src/components/Pressable.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+Pressable is a Core Component wrapper that can detect various stages of press interactions on any of it's defined children.
+
+## How it works
+
+On an element wrapped by Pressable:
+
+1. onPressIn is called when a press is activated.
+2. onPressOut is called when the press gesture is deactivated.
+
+You can read more about this [here](https://reactnative.dev/docs/pressable).
```
Comment (MoOx) [inline src/components/Pressable.md:16]:
```suggestion
You can read more about this [on the official Pressable documentation](https://reactnative.dev/docs/pressable).
```

File: src/components/Pressable.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+---
+id: components/Pressable
+title: Pressable
+wip: true
```
Comment (MoOx) [inline src/components/Pressable.md:4]:
```suggestion
officialDoc: https://reactnative.dev/docs/pressable
```

wip: is reserved to mostly empty markdown. Here you wrote enough to start :)

File: src/components/Pressable.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+Pressable is a Core Component wrapper that can detect various stages of press interactions on any of it's defined children.
+
+## How it works
+
+On an element wrapped by Pressable:
+
+1. onPressIn is called when a press is activated.
+2. onPressOut is called when the press gesture is deactivated.
```
Comment (MoOx) [inline src/components/Pressable.md:14]:
```suggestion
1. `onPressIn` is called when a press is activated.
2. `onPressOut` is called when the press gesture is deactivated.
```

File: src/components/Pressable.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+id: components/Pressable
+title: Pressable
+wip: true
+---
+
+Pressable is a Core Component wrapper that can detect various stages of press interactions on any of it's defined children.
+
+## How it works
+
+On an element wrapped by Pressable:
```
Comment (MoOx) [inline src/components/Pressable.md:11]:
```suggestion
On an element wrapped by `Pressable`:
```

**Review by MoOx — APPROVED** [review]
I adjusted a few thing in doc, but that's more than enough! Thanks again for your contribution!

**Review by MoOx — APPROVED** [review]

**Pull request merged**